src/injected/content/inject.js

```diff
@@ -40,14 +50,19 @@ export function triageScripts(data) {
       return false;
     });
     let support;
-    data.scripts.forEach((script) => {
-      let injectInto = script.custom.injectInto || script.meta.injectInto || data.injectInto;
-      if (injectInto === INJECT_AUTO) {
+    const injectChecking = {
+      [INJECT_INTERNAL_PAGE]: () => {
```

Comment:
This should not be needed in Chrome because script elements created by extensions are explicitly allowed regardless of page CSP.

Reply:
It's true but I think it's safer to check it this way than to check browser type with user-agent or some special properties. This will keep working even someday Chrome changes its behavior.